src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/ChangeRequestsTab.tsx

```diff
@@ -0,0 +1,101 @@
+import { ChangeRequest, WorkPackage } from 'shared';
+import { Box, Grid, Typography, useTheme } from '@mui/material';
+import ChangeRequestDetailCard from '../../../components/ChangeRequestDetailCard';
+import { useAllChangeRequests } from '../../../hooks/change-requests.hooks';
+import LoadingIndicator from '../../../components/LoadingIndicator';
+import ErrorPage from '../../ErrorPage';
+import { projectWbsPipe } from '../../../utils/pipes';
+interface ChangeRequestsTabProps {
+  workPackage: WorkPackage;
+  dependencies: WorkPackage[];
+}
+
+const ChangeRequestsTab: React.FC<ChangeRequestsTabProps> = ({ workPackage, dependencies }) => {
+  const theme = useTheme();
+  const { data: changeRequests, isError: crIsError, isLoading: crIsLoading, error: crError } = useAllChangeRequests();
+  const currentDate = new Date();
+
+  if (crIsLoading || !changeRequests) return <LoadingIndicator />;
+  if (crIsError) return <ErrorPage message={crError?.message} />;
+
+  const crUnreviewed = changeRequests
+    ? changeRequests
+        .filter((cr: ChangeRequest) => !cr.dateReviewed && projectWbsPipe(cr.wbsNum) === projectWbsPipe(workPackage.wbsNum))
+        .sort((a, b) => b.dateSubmitted.getTime() - a.dateSubmitted.getTime())
+    : [];
+
+  const crApproved = changeRequests
+    ? changeRequests
+        .filter(
+          (cr: ChangeRequest) =>
+            cr.dateReviewed &&
+            cr.accepted &&
+            currentDate.getTime() - cr.dateReviewed.getTime() <= 1000 * 60 * 60 * 24 * 5 &&
+            projectWbsPipe(cr.wbsNum) === projectWbsPipe(workPackage.wbsNum)
+        )
+        .sort((a, b) => (a.dateReviewed && b.dateReviewed ? b.dateReviewed.getTime() - a.dateReviewed.getTime() : 0))
+    : [];
+
+  const displayCRCards = (crList: ChangeRequest[]) => (
```

Comment:
This entire function is the same as in the `ChangeRequestOverview.tsx` file. I would say to abstract this function to a file like Change request utils. @RChandler234 is there a different file that would make sense to abstract something like this to?

Reply:
yup @AShyamkumar19, I would copy over the component Jack made in this PR for rows, you should be able to use it here: https://github.com/Northeastern-Electric-Racing/FinishLine/pull/2168